tools/stores_l10n/add_new_product.md

```diff
@@ -0,0 +1,261 @@
+# How to add a new product to Stores
+
+Before starting, make sure to read the [overview](overview.md) document to get a general idea of what this web app does and its structure.
+
+This document will explain how to start tracking a new project, using Focus for Android as an example.
+
+## Add new product to configuration
+
+Configuration for the Stores web app is in the `Project` class (`app\classes\Stores\Project.php`). The first thing to do it to add the project to `$products_data`, and you’ll need to determine beforehand:
+* How many channels are supported.
+* Code name and displayed name for this product.
+* If the product needs to support App Store (apple) or Play Store (google).
+
+This is the configuration for Focus for Android:
+
+```PHP
+private $products_data = [
+...
+    'focus_android' =>
+        [
+            'channels' => ['release'],
+            'name'     => 'Focus for Android',
+            'store'    => 'google',
+        ],
+...
+];
+```
+
+Then you need to add an entry in `$supported_locales` for each of the channels available. Since Focus for Android only has a release channel, it will be:
```

Comment:
Good point. It's shipping locale, as in locales that are available in the app. Will udpate.